### ISSUE TITLE
Internal User: View and Search Reports by Status

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-reporting#96]
As an Internal CG user 
I want to view the status of reports (i.e. "draft" or "submitted") 
So that I can know what reports are still in progress by Industry Reporters

[Have ability to search by draft status to see if an operator has applications in draft for CAS staff.]

**Comment by patriciarussellCAS** [comment]
To confirm: Is it useful for internal staff to see that a report has been started or not? How complicated would this be given some operators have more than one facility report? 
Would this be helpful for external users so they know what has been completed or not? 

Should this be MVP? Not sure.

**Comment by NicoleGovvy** [comment]
@patriciarussellCAS yeah I see how this one can be tricky potentially, it would depend on how statuses are designed. This was to have the ability to search within BCEIRS instead of needing to go to MB to get the info.

**Comment by zoeyli-46** [comment]
@patriciarussellCAS would this be in metabase where they can see non-submitted reports? Based on our last conversation with the BA it seems they only need to see "submitted reports". User stories for the internal user reporting view are in the ticket #497

**Comment by patriciarussellCAS** [comment]
Ack, sorry, responded to this in the other ticket. I still think this would be a nice feature improvement even if they can see this in Metabase, but we should probably revisit after we go live.
